### ISSUE TITLE
代码编译失败

### ISSUE THREAD

**Issue by Hello-jeep** [polywock/globalSpeed#211]
![image](https://user-images.githubusercontent.com/63323127/170520691-29fbb934-5fbd-41a6-b351-4e7c734ec371.png)

**Comment by polywock** [comment]
Hello. What steps did you take to compile it? I tried recently with no errors.

**Comment by Hello-jeep** [comment]
下载代码之后，执行npm install，然后执行npm run build:dev就报错了

**Comment by polywock** [comment]
I think I figured out the problem. You are running Linux, right? I run on Mac and Mac's filesystem isn't case sensitive, unlike Linux. In `src/options/SectionEditor.tsx`, there's an import statement 

```
import { KeybindControl } from "./KeybindControl"
``` 

I mistyped the file, it's actually `KeyBindControl.tsx` (b is capitalized). On Mac it doesn't matter, so I never noticed. On Linux, it matters so it needs to be imported correctly.
